e2e/config/named.conf

```diff
@@ -0,0 +1,47 @@
+options {
```

Comment:
maybe simple dnsmasq daemon would be enough?

Reply:
yea, tested, it works with dnsmasq too. Since its smaller footprint, will switch to it. Thanx.